src/execution_plan/ops/op.c

```diff
@@ -46,7 +53,7 @@ void OpBase_Init
 	op->init     = init;
 	op->free     = free;
 	op->clone    = clone;
-	op->reset    = reset;
+	op->reset    = reset ? reset : _OpBase_reset_noop;
```

Comment:
a simple modification, make it a bit easier for me to read:
`(reset) ? reset : _OpBase_reset_noop;`